README.md

```diff
@@ -35,6 +35,40 @@ make
 
 This builds libmongocrypt.dylib and test-libmongocrypt, in the cmake-build directory. Note, the `CMAKE_PREFIX_PATH` must include the path to the BSON library installation directory if it was not the defaults.  Also note that if your project will also dynamically link to the BSON library, you will need to add `-DENABLE_SHARED_BSON=ON` to the `cmake` command line.
 
+## Installing libmongocrypt on macOS ##
+First install [Homebrew according to its own instructions](https://brew.sh/). Using Homebrew, install the following dependencies.
+```
+brew install mongo-c-driver cmake
+```
+
+Install the XCode Command Line Tools:
+```
+xcode-select --install
+```
+
+Then clone and build libmongocrypt:
+```
+git clone git@github.com:mongodb/libmongocrypt.git
```

Comment:
I recall that during last week's meeting someone (I forget who) pointed out that cloning over https would prevent certain users getting errors if there was no ssh key configured in GitHub.  That point was made about the compile script that cloned and build libbson.  However, I think that for documentation meant for general public consumption (as opposed to active GitHub contributors), it is good practice to keep to https URLs.

Reply:
Good point, done.